### DISCUSSION TITLE
Power adjustment for Matek R24

### DISCUSSION THREAD

**Discussion by AaryanL** [ExpressLRS/ExpressLRS#1375] [Ideas]
Matek R24 has quite strong telemetry power, it's not always needed for close range flights.
Is it possible to add Max power selection in Lua, similar to how it's made for Tx modules?

**Comment by pitts-mo** [comment]
I generally do not like the idea of setting arbitrary limit on the telemetry power but I can see it may be useful in specific builds.

However, I do feel it is desirable for the RX to support dynamic output power for telemetry and max power should remain the max power for the current modulation type in the regulatory domain. 

My initial questions:
Does the TX currently report the telemetry LQ back to RX? Or can this be added to wide mode?

Would there be any good reason to add a max power override as a build time option within the regulatory domain?

**Comment by SunjunKim** [comment]
As telemetry packets are sparse, and it’s totally OK with a short burst of max-power packets. There’s little need for adjusting the power and make the thins complex.

> Does the TX currently report the telemetry LQ back to RX? Or can this be added to wide mode?

No. If we do, it will break OTA packet protocol, and it’s less priority thing.. so it’s not likely to be implemented.

> Would there be any good reason to add a max power override as a build time option within the regulatory domain?

If there’s a very good reason for doing that, limiting the max power in compile time is totally possible with few lines of change in code.

**Comment by StonedDawg** [comment]
linking this with the RX params PR, https://github.com/ExpressLRS/ExpressLRS/pull/1362
